docker/README.md

```diff
@@ -0,0 +1,12 @@
+## Yacguide Docker Build Environment
+
+This is the Docker image for building the `yacguide` Android app.
+
+See <https://github.com/paetz/yacguide> for more information.
```

Comment:
Adapt url (now "YacGroup" instead of "paetz")

Reply:
You are right. I will do so.